gulpfile.js

```diff
@@ -180,11 +181,18 @@ gulp.task('docs:mkdocs', shell.task([
   'mkdocs serve'
 ], {
   cwd: 'docs'
-}))
+}));
+
+gulp.task('lint', function () {
+  return gulp.src(['src/pathfora.js', 'gulpfile.js', 'test/pathforaSpec.js', 'docs/docs/examples/**/*.js'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
 
 gulp.task('test', ['build:styles', 'build:testjs']);
 gulp.task('build:local', ['build:styles', 'local:js']);
```

Comment:
yah it might be annoying in the build task that watches for sure. we/i can just make a new task to run before we merge anything. or maybe we include it with tests only?